community_images/yourls/official/image.yml

```diff
@@ -0,0 +1,39 @@
+name: yourls
+official_name: YOURLS
+official_website: https://yourls.org/
+source_image_provider: YOURLS
+source_image_repo: docker.io/_/yourls
```

Comment:
I think _ should be library here. Please check

Reply:
Rectifying it.